### PR TITLE
Biological NER predictor pack() missing context parameter

### PR DESCRIPTION
**Pull request by feipenghe**
This PR fixes https://github.com/asyml/forte/issues/622. 

### Description of changes
Add `context` parameter to the function to adapt API interface changes even though `context` is not needed.

### Possible influences of this PR.
Indexer can run successfully in the clinical pipeline example 
https://github.com/asyml/forte/tree/master/examples/clinical_pipeline

### Test Conducted
* provide a sample data that contains biological NER in every data packs and assert the number of NER for each data pack larger than zero
* check the number of json pickle file generated is same as variable `self.num_packs`.

### CODE REVIEW

File: src/huggingface/fortex/huggingface/bio_ner_predictor.py [hunk 1]
```diff
@@ -191,15 +197,15 @@ def pack(
         self,
         data_pack: DataPack,
         output_dict: Optional[Dict[str, Dict[str, List[Any]]]] = None,
+        context: Optional[Annotation] = None,
     ):
         """
         Write the prediction results back to datapack. by writing the predicted
-        ner to the original subwords and convert predictions to something that
+        ner to the context and convert predictions to something that
```
Comment (hunterhector) [inline src/huggingface/fortex/huggingface/bio_ner_predictor.py:204]: We simply say something like write predicted results by aggregating subwords into named entity mentions

File: src/huggingface/fortex/huggingface/bio_ner_predictor.py [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 import torch
+import logging
+from importlib import import_module
 from forte.common.configuration import Config
 from forte.common.resources import Resources
 from forte.data.data_pack import DataPack
+from forte.data.ontology.top import Annotation
 from forte.processors.base.batch_processor import RequestPackingProcessor
-from ft.onto.base_ontology import EntityMention, Subword
+from ft.onto.base_ontology import EntityMention, Subword, Token, Sentence
```
Comment (hunterhector) [inline src/huggingface/fortex/huggingface/bio_ner_predictor.py:16]: Should be careful about import orders, this is part of python convention

**Review by hunterhector — COMMENTED** [review]
And remember to add test cases

File: src/huggingface/fortex/huggingface/bio_ner_predictor.py (hunk 2 above)
Comment (feipenghe) [reply src/huggingface/fortex/huggingface/bio_ner_predictor.py]: Do you mean the order of modules after `from ft.onto.base_ontology import` should follow alphabetic order?

File: src/huggingface/fortex/huggingface/bio_ner_predictor.py (hunk 1 above)
Comment (feipenghe) [reply src/huggingface/fortex/huggingface/bio_ner_predictor.py]: fixed it.

File: src/huggingface/fortex/huggingface/bio_ner_predictor.py (hunk 2 above)
Comment (hunterhector) [reply src/huggingface/fortex/huggingface/bio_ner_predictor.py]: https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement

**Review by hunterhector — APPROVED** [review]

**Pull request merged**